Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.cs

```diff
@@ -42,7 +42,7 @@ public void When_type_is_equal_to_another_type_it_fails()
 
             // Act
             Action act = () =>
-                type.Should().Be(differentType, "because we want to test the failure {0}", "message");
```

Comment:
🤔 Do we still have a test that verifies if the `because` is added for you?

Reply:
I would expect every assertion to only test that because and becauseArgs are handled correctly.

The actual correct working of `BecauseOf()` is covered by `When_reason_does_not_start_with_because_it_should_be_added` in [AssertionFailureSpecs](https://github.com/eNeRGy164/fluentassertions/blob/7521863367b7b3bde573505f0da08f3baaae0a10/Tests/FluentAssertions.Specs/AssertionFailureSpecs.cs#L29).